iroha/src/block.rs

```diff
@@ -240,6 +306,64 @@ impl CommittedBlock {
     }
 }
 
+impl From<&CommittedBlock> for Vec<Event> {
+    fn from(block: &CommittedBlock) -> Self {
+        block
+            .transactions
+            .iter()
+            .cloned()
+            .map(|transaction| {
+                PipelineEvent {
+                    entity_type: PipelineEntityType::Transaction,
+                    status: PipelineStatus::Committed,
+                    hash: transaction.hash(),
+                }
```

Comment:
It's worth to create constructor `PipelineEvent::commited_transaction(hash: Hash)`.

Reply:
As there are 3 possible statuses and 2 possible entity types, this would mean that we should have 6 constructors for pipeline events, which seems a bit overcomplicated to me...

Reply:
More statuses might be added in the future, for example for multisignature transactions, then we would have even more constructors this way.